actor/builtin/miner/miner_test.go

```diff
@@ -384,11 +384,58 @@ func TestMinerCommitSector(t *testing.T) {
 		// fail because commR already exists
 		res, err = th.CreateAndApplyTestMessage(t, st, vms, minerAddr, 0, 4, "commitSector", nil, uint64(1), commD, commR, commRStar, th.MakeRandomBytes(types.TwoPoRepProofPartitions.ProofLen()))
 		require.NoError(t, err)
-		require.EqualError(t, res.ExecutionError, "sector already committed")
+		require.EqualError(t, res.ExecutionError, "sector already committed at this ID")
 		require.Equal(t, uint8(0x23), res.Receipt.ExitCode)
 	})
 }
 
+func TestMinerSubmitPoStNextDoneChanges(t *testing.T) {
```

Comment:
I made a new test instead of a new `t.Run` subfunction to try to prevent further complicating the `TestMinerSubmitPoSt` setup.  This seemed worth it to me but I'd appreciate feedback on best practices for actor tests if anything about this seems ugly.